<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveableWallForm</class>
 <widget class="QDialog" name="MoveableWallForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>405</width>
      <height>135</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBoxTriggerAppearence">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>403</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxDirection">
      <item>
       <property name="text">
        <string>NORTH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WEST</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SOUTH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EAST</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Move case number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxMoveNumber">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAddMove">
      <property name="text">
       <string>Add move step</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>421</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Velocity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxVelocity">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxTriggerBehaviourType">
      <item>
       <property name="text">
        <string>ONCE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MULTI REVERSE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MULTI SAME</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AUTO</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxTriggerType">
      <item>
       <property name="text">
        <string>WALL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EXTERN TRIGGER</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GROUND</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
